cluster/ci/config/prow/plugins.yaml

```diff
@@ -31,6 +31,10 @@ config_updater:
       name: ci-operator-kubernetes-descheduler
     ci-operator/config/kubernetes/descheduler/release-3.10.json:
       name: ci-operator-kubernetes-descheduler
+    ci-operator/config/kubernetes/descheduler/master_e2e.json:
+      name: ci-operator-kubernetes-descheduler
+    ci-operator/config/kubernetes/descheduler/release-3.10_e2e.json:
```

Comment:
You shouldn't have to create this - your tests should be in release-3.10.  If you're updating release-3.10 to use e2e, you'd just update that file.